class/Troubadore; Versatile Ranger.json

```diff
@@ -3,13 +3,13 @@
 		"sources": [
 			{
 				"json": "TheVersatileRanger",
-				"abbreviation": "TTVR",
+				"abbreviation": "TVR",
 				"full": "Troubadore's Versatile Ranger",
 				"authors": [
 					"Troubadore"
 				],
 				"convertedBy": [
-					"Matsozetex#8230"
```

Comment:
Leave this name in, they also converted it. And this tag is supposed to be a full list of people who worked on it

Reply:
Ah gotcha, I didn't know! Er, so sorry, but I'm super new to github, so may I ask how to do this? Should I go back to my fork and edit it, making a new PR?

Reply:
I've added it for you. So it's all good now.

Reply:
Thanks so much!